### ISSUE TITLE
Nimbus experiments can be sorted by the Monitoring and Results columns

### ISSUE THREAD

**Issue by heres-maria** [mozilla/experimenter#6126]
**[Prerequisites]:**
- Have the main Nimbus page opened on “Completed” or “Live”.

**[Steps to reproduce]:**
1. Click the “Results” column header.
2. Observe the experiments table.

**[Expected result]:**
- Columns with no discernible value are not used to sort experiments.

**[Actual result]:**
- Columns like “Monitoring” or “Results” can be used to sort experiments.

**[Notes]:**
- Attached is a recording of the issue.

![image](https://user-images.githubusercontent.com/53752296/128363528-68e1f0ae-35a0-4340-a3d0-285d62fde0d7.png)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/EXP-1586)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/secure/attachment/15934/Capture.PNG">Capture.PNG</a>

**Comment by data-sync-user** [comment]
➤ Les Orchard commented:

I think Results is a valid field on which to sort - it can either have results or not, so that can be a useful sort.

!Capture.PNG|width=858,height=518!

**Comment by lmorchard** [comment]
Ended up leaving Results as sortable in the Live view, but disabled in Complete because it is never N/A by that point

**Comment by heres-maria** [comment]
I have verified that the "Results" header is no longer clickable on the "Completed" tab and that the "Monitoring" header is no longer clickable on the "Live" tab. 
Additionally, when you sort Live experiments by "Results" and all the experiments have the same value on that column (e.g. N/A), experiments are no longer reordered. The list remains the same.
Tested using Firefox 91.0 (Build ID: 20210804193234) on Windows 10.
